src/applications/vaos/newAppointmentFlow.js

```diff
@@ -78,6 +83,8 @@ export default {
 
       if (isSleepCare(state)) {
         nextState = 'typeOfSleepCare';
+      } else if (isEyeCare(state)) {
```

Comment:
This is an interesting wrinkle. The way this works right now, you're going to be skipping all the CC eligibility logic, because those checks happen in the following else block. We can get away with that for sleep care because it's never CC eligible.

I _think_ what you'll need to do is create some kind of `isCommunityCareEligible` function that you can run from the `typeOfEyeCare` `next` function, as well as in the else block below this.